PLATEXP-1731: Rackspace Dedicated Hosting - Delete a VPN user in the MyRackspace Portal (#1426)
rackerlabs/support-how-to: content/how-to/dedicated-hosting/delete-vpn-user-myrackspace-portal/index.md

type: article
created_date: '2017-10-16'
created_by: Trevor Becker

> last_modified_date: '2022-09-28'

Reviewer: I need to remember to update the date right before merging

### Create a ticket in the MyRackspace Portal

**Step 1.** Log in to the [MyRackspace Portal](https://login.rackspace.com) by using your
   username and password.

>    <img width="409" alt="Enter Login credential for Username and Password" src="/support/how-to/delete-vpn-user-myrackspace-portal/deletevpn1.png">

Reviewer 2: Images are not visible.

**Step 2.** In the top navigation bar, click **Support > Support Center**.

   <img width="941" alt="Select Support Center from top navigation bar" src="/support/how-to/delete-vpn-user-myrackspace-portal/deletevpn2.png"> 

> **Step 3.** From the **Common Request** menu select **VPN - add, reset, delete VPN user(s) on firewalls** as shown in following image.

Reviewer 2: ```suggestion
**Step 3.** From the **Common Request** menu, select **VPN - add, reset, delete VPN user(s) on firewalls** .
```

Reviewer 2: Merge Step 3&4.

   <img width="830" alt="Select the required template from the common request menu" src="/support/how-to/delete-vpn-user-myrackspace-portal/deletevpn3.png">

**Step 4.** You can view the **Issue Details** section by selecting **VPN - add, reset, delete VPN user(s) on firewalls** as shown in following image.

   <img width="508" alt="Issue details section is viewed by selecting the template" src="/support/how-to/delete-vpn-user-myrackspace-portal/deletevpn4.png">

> **Step 5.** Check the box next to **Delete VPN user on firewall** as shown in following image.

Reviewer 2: Merge Step 5&6

   ensure that you select all firewalls in this field.

   <img width="440" alt="Select the required firewall which needs to tbe deleted" src="/support/how-to/delete-vpn-user-myrackspace-portal/deletevpn6.png">

**Step 7.** Enter your desired VPN username in the **VPN Username** field.

   a. The VPN username is case-sensitive and requires at least three characters.

   b. Do not include a question mark (?) or space ( ) within the VPN username field.

   <img width="423" alt="Enter the username credentials" src="/support/how-to/delete-vpn-user-myrackspace-portal/deletevpn7.png">

**Step 8.** Click **Submit** at the bottom of the page.

   After the ticket generates, you see your ticket list in
   the MyRackspace Portal. Automation runs in the background while the process
   creates the ticket, which typically takes less than a minute to complete.
   After you refresh your browser, the ticket *Delete VPN User on Firewall Request (from template)*
   is in **Solved** status.

**Step 9.** Confirm the details about the deletion of the VPN user by opening the ticket.
   The details are in the most recent comment.

> If you need any assistance outside of this article, open a ticket so that

Reviewer 2: ```suggestion
If you need any assistance outside of this article, open a ticket and
```

1. Log in to the [MyRackspace Portal](https://login.rackspace.com) by using your
   username and password.

>    <img width="409" alt="logincredentials" src="/support/how-to/delete-vpn-user-myrackspace-portal/deletevpn1.png">

Reviewer: Fix alt tags on images

Author: Done

2. In the top navigation bar, click **Support > Support Center**.

   <img width="941" alt="selectsupport" src="/support/how-to/delete-vpn-user-myrackspace-portal/deletevpn2.png"> 

3. From the **Common Request** menu select **VPN - add, reset, delete VPN user(s) on firewalls** as shown in following image.

   <img width="830" alt="selectdeletevpn" src="/support/how-to/delete-vpn-user-myrackspace-portal/deletevpn3.png">

> 4. You can view **Issue Details** section by selecting **VPN - add, reset, delete VPN user(s) on firewalls** as shown in following image.

Reviewer: add `the` before issues details.. for example `You can view the **Issue Details** section`

Author: Done

   b. If you need to delete the VPN user from all firewalls on the account,
   ensure that you select all firewalls in this field.

>    <img width="410" alt="selectfirewall" src="/support/how-to/delete-vpn-user-myrackspace-portal/deletevpn6.png">

Reviewer: This image has production device IDs, names and numbers showing. We probably want to manipulate the browser in the inspector and change the info before taking screenshot. I can show how to do that if needed.

Author: Image is changed as suggested.

7. Enter your desired VPN username in the **VPN Username** field.

   a. The VPN username is case-sensitive and requires at least three characters.

   b. Do not include a question mark (?) or space ( ) within the VPN username field.

   <img width="423" alt="usernamecredential" src="/support/how-to/delete-vpn-user-myrackspace-portal/deletevpn7.png">

8. Click **Submit** at the bottom of the page.

   After the ticket generates, you see your ticket list in
   the MyRackspace Portal. Automation runs in the background while the process
   creates the ticket, which typically takes less than a minute to complete.
   After you refresh your browser, the ticket *Delete VPN User on Firewall Request (from template)*

>    is in **Solved** status.

Reviewer: So it went to `Solved` status first? If so, I think that is correct. Customer needs to close it and then it goes to Closed I think. After several days of being open I think it will autoclose.

Author: Yes, checked.
